mllib/src/main/scala/org/apache/spark/mllib/package-info.java

```diff
@@ -16,6 +16,26 @@
  */
 
 /**
- * Spark's machine learning library.
+ * RDD-based machine learning APIs (in maintenance mode).
+ *
+ * The <code>spark.mllib</code> package is in maintenance mode as of the Spark 2.0.0 release to
+ * encourage migration to the DataFrame-based APIs under the <code>spark.mllib</code> package.
```

Comment:
Ah, I made this typo while playing with JavaDoc tags ... I tried to link to the `spark.ml` Java package doc but no success and ended up with `<code>`.